src/Microsoft.Framework.PackageManager/Publish/PublishManager.cs

```diff
@@ -140,13 +143,9 @@ public bool Publish()
                     var runtimeHome = Environment.GetEnvironmentVariable(EnvironmentNames.Home);
```

Comment:
The logic here requires that we use environment variable here (if `DNX_HOME` is set do ..., otherwise ...). So there is no `DnuEnvironment.GetFolderPath()` for this.